### ISSUE TITLE
Add description to Locations

### ISSUE THREAD

**Issue by AlanDeSmet** [wlonk/wheretofind.me#93]
Currently if I want to add context to a Location, I need to stuff it into the Name or Tag. The Tag doesn't seem like the right place for extended text, and putting it in the Name is just unwieldy. It would be nice to have an optional field for a Description. It should probably be displayed smaller or lighter than the Name.

![wtfm-proposed-description](https://user-images.githubusercontent.com/610177/52320111-0d34b480-2993-11e9-8f89-97b18b209fb3.png)

**Comment by wlonk** [comment]
I like this idea. I think we can throw in an additional field in the expanded-card area, for those who want commentary like this on a link. If you don't get a PR in for this, I'll add it to my backlog and try to implement it myself.

**Comment by Z33DD** [comment]
I'm a backend developer, can I help to build this feature in any way?

**Comment by ojacobson** [comment]
Absolutely. We're open to contributions. The relevant code starts in [models.py](https://github.com/wlonk/wheretofind.me/blob/main/wheretofindme/models.py#L228), with the UI largely in [identity.vue](https://github.com/wlonk/wheretofind.me/blob/main/src/components/Identity.vue) and [user_detail.html](https://github.com/wlonk/wheretofind.me/blob/main/templates/wheretofindme/user_detail.html), though don't treat this as an exhaustive list.

Licensing and rights around the code are a bit of a ??? right now, and we haven't taken on substantial outside contributions yet, so this will be a bit of an exercise. If you contribute, would you be comfortable with us distributing your work and allowing others to modify it in the future?

**Comment by Z33DD** [comment]
I like where to find me and I'll love to help. But I really need help because I have almost zero frontend development skills.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
